experimental/algorithm/LAGr_MarkovClustering.c

```diff
@@ -124,81 +136,81 @@ int LAGr_MarkovClustering(
         // Normalization step: Scale each column in T_temp to add up to 1
         // w = 1 ./ sum(A(:j))
         // D = diag(w)
-        GRB_TRY(GrB_reduce(w, NULL, NULL, GrB_PLUS_MONOID_FP32, T_temp, GrB_DESC_RT0));
-        GRB_TRY(GrB_apply(w, NULL, NULL, GrB_MINV_FP32, w, GrB_DESC_R));
+        GRB_TRY(GrB_reduce(w, NULL, NULL, GrB_PLUS_MONOID_FP32, T_temp,
+                           GrB_DESC_T0));
+        GRB_TRY(GrB_apply(w, NULL, NULL, GrB_MINV_FP32, w, NULL));
+        if (D != NULL) GrB_free(&D);
```

Comment:
Not sure if this is necessary, I was getting memory leaks if this check was not done though